README.md

```diff
@@ -30,6 +30,7 @@ The main goal is to use modern sensors (mainly [I2C](https://github.com/orgua/iL
 - **DS2505 (0x0B) 16kbit EEPROM, Add Only Memory** (also known as DS1985, same FC) -> use DS2506 with different family code
 - **DS2506 (0x0F) 64kbit EEPROM, Add Only Memory** (also known as DS1986, same FC)
 - **DS2890 (0x2C) Single channel digital potentiometer - extended to 1-4 CH**
+- **DellAC (0x28) Original Dell notebook charger emulator **
 - Dell Power Supply (use DS2502 with family code set to 0x28)
```

Comment:
you see line 34, right? the dell power supply is already included, no need to double the work